hoodie-client/src/main/java/com/uber/hoodie/io/HoodieCleanHelper.java

```diff
@@ -48,6 +55,7 @@
 
   private final TableFileSystemView fileSystemView;
   private final HoodieTimeline commitTimeline;
+  private final Map<String, HoodieCompactionOperation> fileIdToPendingCompactionOperations;
```

Comment:
Can we avoid defining this as a member variable ? Seems out of place...

Reply:
We can revisit refactoring later.